src/gluonnlp/model/bert.py

```diff
@@ -665,6 +703,50 @@ def bert_24_1024_16(dataset_name=None, vocab=None, pretrained=True, ctx=mx.cpu()
                           pretrained_allow_missing=pretrained_allow_missing, **kwargs)
 
 
+def ernie_12_768_12(dataset_name=None, vocab=None, pretrained=True, ctx=mx.cpu(),
+                    root=os.path.join(get_home_dir(), 'models'), use_pooler=True, use_decoder=True,
+                    use_classifier=True, **kwargs):
+    """baidu ERNIE model.
+
```

Comment:
Could you add a link to their paper: https://arxiv.org/pdf/1904.09223.pdf

Reply:
added